src/store/modules/user.ts

```diff
@@ -71,28 +68,28 @@ export class UserStore extends VuexModule implements User {
       .doc(user.uid)
       .get()
 
-    const allowAccessData: AllowAccessFromUsers[] = data.get('allow_access')
+    const allowAccessData: string[] = data.get('allow_access')
     const allowAccess = []
```

Comment:
```suggestion
    const allowAccessClassIds: string[] = data.get('allow_access')
    const allowAccessData: AllowAccessData[] = []
```
データ構造的にはこっちの方が誤読しなさそう（loginFromUserObjectでも同じ）